example/fedora/minimal-40-aarch64.yaml

```diff
@@ -1,7 +1,14 @@
 otk.version: 1
 
 otk.define:
-  version: 40
-  architecture: "aarch64"
+  otk.op.join:
+    - version: 40
+      architecture: "aarch64"
+    - otk.include: "minimal-common.yaml"
 
-otk.include: "minimal-common.yaml"
+otk.target.osbuild:
```

Comment:
I wonder if we can do better here, not including the pipelines from something like "common.yaml" will mean we have quite a bit of duplication.

Reply:
We could resolve includes before omnifest validation.